src/metrics/list_repos.rs

```diff
@@ -125,3 +126,41 @@ async fn count_pull_requests(org_name: &String, repo_name: &str) -> usize {
 
     pr_count
 }
+
+#[derive(Deserialize, Debug)]
+struct Response<T> {
+    data: T,
+    errors: Option<Vec<serde_json::Value>>,
+}
+
+#[derive(Deserialize, Debug)]
+struct Data {
+    search: IssueCount,
+}
+
+#[derive(Deserialize, Debug)]
+struct IssueCount {
+    #[serde(rename = "issueCount")]
+    issue_count: u32,
+}
+
+#[throws]
+async fn count_pull_requests_graphql(org_name: &str, repo_name: &str) -> usize {
+    let octo = octocrab::instance();
+    let s = format!("{}", (Utc::now() - Duration::days(30)).date());
+    let mut chars = s.chars();
+    chars.next_back();
+    chars.next_back();
+    chars.next_back();
```

Comment:
what on earth is this btw :) I guess shaving off some characters?

Reply:
there has to be a better way...but at least it needs a comment :)

Reply:
haha SO fair.  I'm just trying to get the Date string in the right format. 

It formats like `"2021-05-18UTC"` but we need it like `"2021-05-18"` so I'm removing three chars from the end. 


Very open to suggestions for improvement on how to do this 😆

Reply:
FWIW since you’re already using chrono you can have it parse it from the string.

```rust
chrono::NaiveDate::parse_from_str(input, "%FUTC") // %F == %Y-%m-%d

Reply:
That's perfect @XAMPPRocky thank you!